src/cmd/Makefile.am

```diff
@@ -20,6 +20,7 @@ fluxcmd_ldadd = \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libflux-core.la \
 	$(top_builddir)/src/common/libflux-optparse.la \
+        $(top_builddir)/src/shell/libmpir.la \
```

Comment:
`libmpir.la` should be added to only the `flux-job` command, not all commands. (See `flux_start_LDADD` for an example of how to do this)

Reply:
Yes you are right. Sorry about that.